dask/bag/tests/test_bag.py

```diff
@@ -42,6 +42,12 @@
 b = Bag(dsk, "x", 3)
 
 
+@pytest.fixture(autouse=True)
+def no_convert_string():
+    with dask.config.set({"dataframe.convert_string": False}):
+        yield
```

Comment:
I see this being used in a few places. It'd be nice if we could define this centrally in one place